src/components/BottomSheet/components/BottomSheetBackdrop.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {memo, useMemo} from 'react'
+import {TapGestureHandler, TapGestureHandlerGestureEvent} from 'react-native-gesture-handler'
+import Animated, {
+  runOnJS,
+  useAnimatedGestureHandler,
+  useAnimatedStyle,
+  withTiming,
+} from 'react-native-reanimated'
+import styled from 'styled-components'
+import {useBottomSheet} from '../hooks'
+import type {ITheme} from 'src/theme'
+import type {StyleProp, ViewStyle} from 'react-native'
+
+const Container = styled(Animated.View)<{theme: ITheme}>(({theme}) => ({
+  position: 'absolute',
+  top: 0,
+  left: 0,
+  right: 0,
+  bottom: 0,
```

Comment:
You shoud use `StyleSheet.AbsoluteFillStyle` for these

Reply:
updated